_episodes/02-shell_to_cwl.md

```diff
@@ -1,47 +1,513 @@
 ---
-title: "Shell to CWL workflow conversion"
+title: "CWL and Shell Tools"
 teaching: 0
 exercises: 0
 questions:
 - "What is the difference between a CWL tool description and a CWL workflow?"
 - "How can we create a tool descriptor?"
 - "How can we use this in a single step workflow?"
-- "How can we expand to a multi-step workflow?"
 objectives:
-- "dataflow objectives:"
-- "explain the difference between a CWL tool description and a CWL workflow"
+#- "dataflow objectives:"
+# "explain the difference between a CWL tool description and a CWL workflow"
 - "describe the relationship between a tool and its corresponding CWL document"
 - "exercise good practices when naming inputs and outputs"
-- "Be able to make understandable and valid names for inputs and outputs (not 'input3')"
-- "describing requirements objectives:"
-- "identify all the requirements of a tool and define them in the tool description"
-- "use `runtime` parameters to access information about the runtime environment"
-- "define environment variables necessary for execution"
-- "use `secondaryFiles` or `InitialWorkDirRequirement` to access files in the same directory as another referenced file"
-- "use `$(runtime.cores)` to define the number of cores to be used"
-- "use `type: File`, instead of a string, to reference a filepath"
-- "converting shell to cwl objectives:"
-- "identify tasks, and data links in a script"
-- "recognize loops that can be converted into scatters"
-- "finding and reusing existing CWL command line tool descriptions"
-- "capturing outputs objectives:"
+## "Be able to make understandable and valid names for inputs and outputs (not 'input3')"
+# "describing requirements objectives:"
+# "identify all the requirements of a tool and define them in the tool description"
+# "use `runtime` parameters to access information about the runtime environment"
+# "define environment variables necessary for execution"
+# "use `secondaryFiles` or `InitialWorkDirRequirement` to access files in the same directory as another referenced file"
+# "use `$(runtime.cores)` to define the number of cores to be used"
+## "use `type: File`, instead of a string, to reference a filepath"
+- "understand how to reference files for input and output"
+#- "converting shell to cwl objectives:"
+# "identify tasks, and data links in a script"
```

Comment:
Can we have a companion PR against the intermediate repo that moves stuff over?

And then you can delete lines instead commenting them out